shared/util/feature-flags.native.tsx

```diff
@@ -24,7 +24,7 @@ const ff: FeatureFlags = {
   stellarExternalPartners: true,
   tabletSupport: true,
   teamInvites: false,
-  teamsRedesign: false,
+  teamsRedesign: true,
```

Comment:
ditto, probably shouldn't flip this on just yet

Reply:
Oops, this one was a mistake